vcpkg/concepts/default-features.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Understanding default features in vcpkg
+description: This article explains the concept of default features, their capabilities, and their expected contents in detail.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 02/01/2024
+ms.topic: concept-article
+---
+
+# Concept: Default features
+
+This article delves into the concept of default features, their implications, and strategies for effective management.
+
+Default features in vcpkg are a set of features automatically included when a port is installed, unless explicitly disabled. These features ensure that a library maintains a baseline level of functionality, even as the complexity of its dependency graph increases.
```

Comment:
```suggestion
Default features in vcpkg are a set of features automatically included when a port is installed, unless explicitly disabled. These features ensure that a library maintains a baseline level of functionality, even as the complexity of its dependency graph increases.
```

We need a Features article to link to. My first thought when reading "Default features in vcpkg are a set of features" was "what is a feature?"